doc/source/api/solver/fielddata.rst

```diff
@@ -45,65 +47,99 @@ the ``get_surface_data`` method and specifying ``Vertices`` for ``data_type``.
 
   >>> from ansys.fluent.core.services.field_data import SurfaceDataType
 
-  >>> field_data.get_surface_data(surface_name="inlet", data_type=SurfaceDataType.Vertices)
-  {4: array([-0.34760258,  0.        , -0.04240644, ..., -0.2953132 ,
-        0.        , -0.06207064], dtype=float32)}
+  >>> vertices_data = field_data.get_surface_data(surface_name="cold-inlet", data_type=SurfaceDataType.Vertices)
+  >>> vertices_data.size
+  241
+  >>> vertices_data.surface_id
+  3
+  >>> vertices_data[5].x
+  -0.2
+  >>> vertices_data[5].y
+  -0.10167995
+  >>> vertices_data[5].z
+  0.0036200774
+
+You can call the same method to get the corresponding surface face normals, and centroids
+by specifying ``FacesNormal`` and ``FacesCentroid`` respectively for ``data_type``.
 
+.. code-block:: python
+
+  >>> faces_normal_data = field_data.get_surface_data(
+  >>>     data_type=SurfaceDataType.FacesNormal, surface_name="cold-inlet"
+  >>> )
 
-You can call the same method to get the corresponding surface face connectivity, normals, and centroids
-by specifying ``FacesConnectivity``, ``FacesNormal`` and ``FacesCentroid`` respectively for ``data_type``.
+  >>> faces_normal_data = field_data.get_surface_data(
+  >>>     data_type=SurfaceDataType.FacesCentroid, surface_name="cold-inlet"
+  >>> )
+
+You can request face connectivity data for a given ``surface_name`` by calling
+the ``get_surface_data`` method and specifying ``FacesConnectivity`` for ``data_type``.
 
 .. code-block:: python
 
-  >>> field_data.get_surface_data(surface_name="inlet", data_type=SurfaceDataType.FacesConnectivity)
-  {4: array([  4,   3,   2, ..., 379, 382, 388])}
+  >>> faces_connectivity_data = field_data.get_surface_data(
+  >>>     data_type=SurfaceDataType.FacesConnectivity, surface_name="cold-inlet"
+  >>> )
+  >>> faces_connectivity_data[5].node_count
+  4
+  >>> faces_connectivity_data[5].node_data
+  [12, 13, 17, 16]
 
-  >>> field_data.get_surface_data(surface_name="inlet", data_type=SurfaceDataType.FacesNormal)
-  {4: array([ 0.0000000e+00,  2.5835081e-06,  ...,
-          2.7459005e-06,  0.0000000e+00,  0.0000000e+00,  3.0340884e-06], dtype=float32)}
-  
-  >>> field_data.get_surface_data(surface_name="inlet", data_type=SurfaceDataType.FacesCentroid)
-  {4: array([-0.34724122,  0.        , -0.0442204 , -0.34724477, ...,
-         -0.04050309, -0.34645557,  0.        , -0.04421487, -0.34646705], dtype=float32)}
-       
-Response contains a dictionary of surface IDs to numpy array of the requested field. 
+
+If a surface name is provided as input response contains, face vertices, connectivity, normal, or centroid data.
+If surface IDs are provided as input response contains, a dictionary containing a map of surface IDs to face
+vertices, connectivity, normal, or centroid data..
```

Comment:
```suggestion
If a surface name is provided as an input, the response contains face vertices, connectivity data, and normal or centroid data.
If surface IDs are provided as the input, the response is a dictionary containing a map of surface IDs to face
vertices, connectivity data, and normal or centroid data.
```
I'm assuming that the changes I made are what you mean. Please revise if incorrect!